_component_design/table-control.md

```diff
@@ -0,0 +1,427 @@
+---
+title: Table Controls
+parent: Tables
+layout: component
+category: Components
+usage: >
+  Table controls are elements that allow users to interact with the data contained in a table. These controls include elements such as search bars, buttons, pagination controls, and action cogs among others. Table controls appear at the top and the bottom of the table and in the table header and table rows.
+preview-image: preview-images/tables-controls.svg
+status: stable
+resource: true
+last-modified: 2018-11-26
+helix-ui-css: false
+helix-ui-javascript: false
+---
+
+{% include toc.html %}
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use controls when you want the user to interact with table data.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
+Consider the following best practices when you construct a table control:
+
+- On the top of the table, a filter button is always the last button in a button group.
+- If you add checkboxes to table rows, list the checkbox first. If you include a cog and not a checkbox, list the cog first.
+
```

Comment:
Add some bp on where to place things, like the search bar. upper left of the table, preferred.
Don't have controls appear or disappear based on the scope of the data in the table.

Reply:
done.